### PR TITLE
Add updated homework file for Glass Data Set

### PR DESCRIPTION
**Pull request by jenderful**
adding updated/amended homework file

### CODE REVIEW

**Comment by justmarkham** [comment]
@jenderful Nice job! I'll add some comments and then merge it in.

File: jennifer/GlassDatasetHomework.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+glass = pd.read_table("http://archive.ics.uci.edu/ml/machine-learning-databases/glass/glass.data", sep=',')
+
+glass.head(10)
+
+#name the columns in the data set
+g_cols = ['id_number', 'RI', 'Na', 'Mg', 'Al', 'Si', 'K', 'Ca', 'Ba', 'Fe', 'Glass_Type']
+glass = pd.read_table('http://archive.ics.uci.edu/ml/machine-learning-databases/glass/glass.data', header=None, sep=',', names=g_cols, index_col='id_number')
+
+#convert into a dataframe
+glass = pd.DataFrame(glass, columns=g_cols)
```
Comment (justmarkham) [inline jennifer/GlassDatasetHomework.py]: read_table will actually return a pandas DataFrame automatically, so there's no need to explicitly convert it into a DataFrame. No harm done, however!

File: jennifer/GlassDatasetHomework.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+# read in data file from url
+pd.read_table("http://archive.ics.uci.edu/ml/machine-learning-databases/glass/glass.data")
+glass = pd.read_table("http://archive.ics.uci.edu/ml/machine-learning-databases/glass/glass.data", sep=',')
+
+glass.head(10)
+
+#name the columns in the data set
+g_cols = ['id_number', 'RI', 'Na', 'Mg', 'Al', 'Si', 'K', 'Ca', 'Ba', 'Fe', 'Glass_Type']
+glass = pd.read_table('http://archive.ics.uci.edu/ml/machine-learning-databases/glass/glass.data', header=None, sep=',', names=g_cols, index_col='id_number')
```
Comment (justmarkham) [inline jennifer/GlassDatasetHomework.py]: Great thinking, to set the id_number as the index. That way, you won't accidentally treat it like a feature.

File: jennifer/GlassDatasetHomework.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+from sklearn import metrics
+print metrics.accuracy_score(y_test, preds)
+
+#Print confusion matrix
+print metrics.confusion_matrix(y_test, preds)
+
+#Calculate accuracy and compare it to the null accuracy rate
+print metrics.accuracy_score(y_test, preds)
+y_test.mean()
+1 - y_test.mean()
```
Comment (justmarkham) [inline jennifer/GlassDatasetHomework.py]: Just keep in mind that whichever of these two numbers is bigger -- y_test.mean() or 1 minus y_test.mean() -- will be your null accuracy rate.

**Comment by jenderful** [comment]
Cool, thanks!

**Pull request merged**